server/comms/server.go

```diff
@@ -448,6 +449,25 @@ func (s *Server) Run(ctx context.Context) {
 		rr.Use(s.limitRate)
 		rr.Get("/config", routeHandler(msgjson.ConfigRoute))
 		rr.Get("/spots", routeHandler(msgjson.SpotsRoute))
+		rr.Get("/logo", func(w http.ResponseWriter, r *http.Request) {
+			handler := httpRoutes[msgjson.LogoRoute]
+			if handler == nil {
+				writeServerError(w, map[string]string{"error": "no handler"})
+				return
+			}
+
+			logo, err := handler(r.Context().Value(ctxThing))
+			if err != nil { // err will always be nil, but this might change.
+				writeServerError(w, map[string]string{"error": err.Error()})
+				return
+			}
```

Comment:
I do think that we should inform the client about the **availability** of a logo in the `msgjson.ConfigResult`, so they can avoid the request altogether if we don't have a file.

Reply:
The logo business will begin life deprecated, because it'll have no place in a mesh.  That's why I just don't think we should bother the important message payloads.  That said, I don't really mind what happens as long as we don't add kilobytes to the config response for this.  An extra http get with a 404 is fine imo, when there is a server that doesn't have a logo, when compared against technical baggage.  If that slows down page rendering, that says this was not engineered correctly anyway because this feature should add no latency in any place but the loading of the logo itself.